src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. Instead, you should create a single instance of `Lottery` and use it to generate all random balls.